### PR TITLE
KM601 - 👌 Remove delete postgres

### PR DESCRIPTION
**Pull request by haavardeide**
## Description

Remove `delete postgres` command, no longer in use via CLI commands

## Motivation and Context

Postgres should be created/removed via cluster.yaml

## How to prove the effect of this PR?

Running `okctl delete postgres` should result in help-text for the `delete` command

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I have updated the release notes (for the next release).

### CODE REVIEW

File: docs/release_notes/0.0.93.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ KM303 ✅ Add delete application (#910)
 KM615 🐛 Fix kubeconfig not using absolute path to aws-iam-authenticator (#924)
 
 ## Changes
+KM601 👌 Remove delete postgres command
```
Comment (deifyed) [inline docs/release_notes/0.0.93.md:12]:
```suggestion
KM601 👌 Remove delete postgres command (#926)
```

**Comment by deifyed** [comment]
Did you verify no usage in Grafana?

**Comment by haavardeide** [comment]
> Did you verify no usage in Grafana?

We don't have metrics for `delete postgres`

**Review by deifyed — APPROVED** [review]

**Comment by deifyed** [comment]
My bad >.>

**Pull request merged**